bigbluebutton-html5/imports/ui/components/user-list/chat-list-item/component.jsx

```diff
@@ -38,26 +43,39 @@ class ChatListItem extends Component {
     } = this.props;
 
     const linkPath = [PRIVATE_CHAT_PATH, chat.id].join('');
+    const isCurrentChat = chat.id === openChat;
 
     let linkClasses = {};
-    linkClasses[styles.active] = chat.id === openChat;
+    linkClasses[styles.active] = isCurrentChat;
 
     if (chat.name === 'Public Chat') {
       chat.name = intl.formatMessage(intlMessages.titlePublic);
     }
 
     return (
       <li className={cx(styles.chatListItem, linkClasses)}>
-        <Link to={linkPath} className={styles.chatListItemLink}>
-          {chat.icon ? this.renderChatIcon() : this.renderChatAvatar()}
-          <div className={styles.chatName}>
-            {!compact ? <h3 className={styles.chatNameMain}>{chat.name}</h3> : null }
-          </div>
-          {(chat.unreadCounter > 0) ?
-            <div className={styles.unreadMessages}>
-              <p className={styles.unreadMessagesText}>{chat.unreadCounter}</p>
+        <Link
+          to={linkPath}
+          className={styles.chatListItemLink}
+          role="button"
+          aria-expanded={isCurrentChat}>
+            {chat.icon ? this.renderChatIcon() : this.renderChatAvatar()}
+            <div className={styles.chatName}>
+              {!compact ? <h3 className={styles.chatNameMain}>{chat.name}</h3> : null }
             </div>
-            : null}
+            {(chat.unreadCounter > 0) ?
+              <div className={styles.unreadMessages}>
+                <div className={styles.unreadMessagesText} aria-describedby="newMsgDesc">
+                  {chat.unreadCounter}
+                  <div
+                    id="newMsgDesc"
+                    aria-label={!(chat.unreadCounter === 1)
```

Comment:
Please consider reversing the condition: `if one ? singular : plural` This way it will be simpler and we avoid extra () and !